connector/saml/saml.go

```diff
@@ -154,6 +154,11 @@ func (c *Config) openConnector(logger logrus.FieldLogger) (*provider, error) {
 		return nil, fmt.Errorf("missing required fields %q", missing)
 	}
 
+	// Set EntityIssuer to RedirectURI if it is not set
+	if c.EntityIssuer == "" {
```

Comment:
I don't want to send this if EntityIssuer isn't explicitly set. Issuer is optional and since you stated this wasn't a safe assumption I think it's safer to avoid this default.